components/youcanbook_me/sources/new-cancelled-booking/new-cancelled-booking.mjs

```diff
@@ -0,0 +1,24 @@
+import base from "../common/booking-base.mjs";
+
+export default {
+  ...base,
+  name: "New Cancelled Booking",
+  version: "0.0.1",
+  key: "youcanbook_me-new-cancelled-booking",
+  description: "Emit new event when a booking is cancelled",
+  type: "source",
+  dedupe: "unique",
+  methods: {
+    async emitEvent(event) {
+      if (!event.cancelled) {
+        return;
+      }
```

Comment:
Can we pass a parameter in the request filtering only cancelled bookings instead of fetch all of them?

Reply:
I don't find a parameter to filter this.

Reply:
Have you tried `fields`? https://api.youcanbook.me/docs/index.html#!/Bookings/getViaAdministrationUsingGET

Reply:
Every booking return a `cancelled` field(with `true` or `false`).

I tried use the `cancelled=true` in the `fields` params, but don't worked 😔